profiles/markaspot/themes/ntxuva/js/ntxuva.js

```diff
@@ -482,7 +484,7 @@ var lang_pt = {
                     var req_date  = new Date(req_datetime),
                         req_year  = req_date.getFullYear(),
                         req_month = req_date.getMonth();
-
+//update this to include years for not loaded requests
```

Comment:
if this is a todo, please mark it properly as a todo comment like: `// TODO: <description>`